packages/xo-web/src/common/xo/index.js

```diff
@@ -2732,6 +2732,20 @@ export const deleteGroup = group =>
     noop
   )
 
+export const deleteGroups = groups =>
+  confirm({
+    title: _('deleteGroupsModalTitle', { nGroups: groups.length }),
+    body: <p>{_('deleteGroupsModalMessage', { nGroups: groups.length })}</p>,
+  }).then(
+    () =>
+      Promise.all(
+        map(resolveIds(groups), id => _call('group.delete', { id }))
```

Comment:
Better avoid using [`resolveId(s)`](https://github.com/vatesfr/xen-orchestra/pull/5167#discussion_r461457517) if it isn't necessary.

```js
groups.map(({ id }) => _call('group.delete', { id }))
```